.github/workflows/lighthouse.yml

```
@@ -0,0 +1,30 @@
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+          
+      - name: Install dependencies
+        run: cd frontend && npm install
+        
+      - name: Build
+        run: cd frontend && npm run build
+        
+      - name: Run Lighthouse CI
+        uses: treosh/lighthouse-ci-action@v10
```

actionlint (1.7.7)

the runner of "treosh/lighthouse-ci-action@v10" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)